source/scripts/util/graph.js

```diff
@@ -3,9 +3,9 @@ import { isA11yEnabled } from '../accessibility.js';
 import { ZEROS } from './storage.js';
 
 /* Graph Constants */
```

Comment:
Might consider changing the comment here since we are not declaring them as constants anymore to accomodate the multilingual feature.